zstd_cmake_module/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.20)
+project(zstd_cmake_module)
+
+find_package(ament_cmake REQUIRED)
+
```

Comment:
I may have misunderstood how this scheme should work, but shouldn't we have somewhere at least
```
find_package(zstd REQUIRED)
```
?

Reply:
@sloretz Thank you for the explanation. In this case, we need to make sure that in other places where we were using `zstd_vendor` present `find_package(zstd REQUIRED)`
cc: @ahcorde

Reply:
`mcap_vendor` and `rosba2_compression` already include this